src/int128_native_impl.h

```diff
@@ -59,7 +59,12 @@ static SECP256K1_INLINE void secp256k1_i128_rshift(secp256k1_int128 *r, unsigned
    *r >>= n;
 }
 
+static SECP256K1_INLINE uint64_t secp256k1_i128_to_u64(const secp256k1_int128 *a) {
+   return *a;
```

Comment:
nit, just for clarity:
```suggestion
   return (uint64_t)*a;
```